Code/Java/Fibonacci_Series.java

```diff
@@ -0,0 +1,107 @@
+/*The Fibonacci sequence is a series where the next term is the sum of pervious two terms. The first two terms of the Fibonacci sequence is 0 followed by 1.
+The Fibonacci sequence: 0, 1, 1, 2, 3, 5, 8, 13, 21....
+*/
+package fibonaccinumbers;
+import java.util.Scanner;
+
+/*Method 1 : Using For Loop*/
+class FibonacciExample1{  
+          public static void main(String args[])  
+             {   
+                  Scanner sc = new Scanner(System.in);
+                  System.out.println("Enter count : ");
+                  int count = sc.nextInt();
+                  System.out.println(count);
+                  int t1 = 0;
+                  int t2 = 1;
+                  int t3;
+                  int i;    
+                  System.out.print(t1 +" "+ t2);        //printing 0 and 1    
+    
+                  for(i = 2 ; i < count ; ++i)               //loop starts from 2 because 0 and 1 are already printed    
+                   {    
```

Comment:
Pl align { of for loop
or you can add like 
```suggestion
         for (i = 2; i <= n; i++) {
```